src/rest/handler_template.erl

```diff
@@ -0,0 +1,63 @@
+%%%-------------------------------------------------------------------
+%%% @author pawel
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 24. May 2021 21:49
+%%%-------------------------------------------------------------------
```

Comment:
Te bloki albo usuwaj, albo skonfiguruj sobie IntelliJ tak żeby to jakoś wyglądało / żeby ich nie wstawiał.

(Sprawdź też inne miejsca)

Reply:
Ok, skoro tak uważasz, to to sobie zmienię (w sumie inni tego nie mają, więc słuszna uwaga).